README.md

```diff
@@ -95,3 +95,23 @@ Q: Nothing is being displayed on the curl
 A: Wait a few more seconds, it takes a moment to display the data. If it's more
 than a minute check to see if the queue has data via the web ui http://localhost:15672/
 
+
+## Vagrant
+
+#### Install
+```
+sudo dnf install vagrant vagrant-libvirt
+sudo dnf copr enable dustymabe/vagrant-sshfs
+sudo dnf install vagrant-sshfs
```

Comment:
You could shuffle the copr enable statement to be the first statement, and then you only need one dnf install line that has all three packages. Also, vagrant-libvirt will pull in vagrant, so you can just do vagrant-libvirt and vagrant-sshfs. Just a suggestion.